ee/api/license.py

```diff
@@ -21,7 +23,17 @@ class Meta:
         read_only_fields = ["plan", "valid_until", "max_users"]
 
     def create(self, validated_data: Any) -> Any:
-        return super().create({"key": validated_data.get("key")})
+        response = None
+        try:
+            response = super().create({"key": validated_data.get("key")})
+            report_license_activated(self.context["request"].user.distinct_id, {"key": validated_data.get("key")})
+        except exceptions.APIException as e:
+            capture_exception(e)
+            raise e
```

Comment:
Why `raise` in the case of an APIException but not for a generic Exception?

Reply:
So my _intention_ here was to preserve the current behavior of the endpoint. Based on the old code it looked like we didn't catch API Exceptions, we just raised them. I wasn't sure if there's upstream handling for it so I didn't want to change that behavior.